src/libs/Pusher/EventType.js

```diff
@@ -11,4 +11,6 @@ export default {
     PREFERRED_LOCALE: 'preferredLocale',
     EXPENSIFY_CARD_UPDATE: 'expensifyCardUpdate',
     SCREEN_SHARE_REQUEST: 'screenshareRequest',
+    ONYX_API_UPDATE: 'onyxApiUpdate',
+    ONYX_API_UPDATE_CHUNK: 'chunked-onyxApiUpdate',
```

Comment:
After [these changes](https://github.com/Expensify/App/pull/9203/files#r883952420
) we won't need the prepend chunked to channel names. 

cc @danieldoglas to confirm

Reply:
Oh nice! That one seems fully approved, so it should be merging today, right @alex-mechler?

Reply:
Yup, that sounds fine. I have to remove the other `chunked-` channels later from this repo anyways.